incubator/ro-cert-manager/README.md

```diff
@@ -2,6 +2,16 @@
 
 Installs the necessary additional items that the cert-manager chart lacks.
 
+## Known Issues
+
+### Race Condition With Cert-manager Chart
+
+When this chart is installed too quickly after versions 0.6+ of the cert-manager chart, the cert-manager validating webhook may not yet be initialized, making cert-manager incapable of creating Clusterissuer objects.
+
+Until this chart is updated to address this issue, we recommendseting `cert-manager.enabled` to `false` for this chart, and installing the cert-manager chart separately. Allow one minute for the cert-manager webhook to initialize before installing this chart.
```

Comment:
Minor typo here: "recommendseting"

Reply:
Another very minor clarification: This starts off saying "Until this chart is updated to address this issue" but I think it means "Until the cert-manager chart is updated".